torch/csrc/jit/c10_ops/layer_norm.cpp

```diff
@@ -45,15 +45,17 @@ RegisterOperators reg({
 
         c10::intrusive_ptr<caffe2::Blob> cache = c10::make_intrusive<caffe2::Blob>();
         cache->GetMutable<c10::core::opschema::LayerNorm::Cache>(); // initialize cache
-        c10::Dispatcher<c10::core::opschema::LayerNorm>::call(ArrayRef<c10::IValue>{
+        // TODO remove std::array for args here, instead pass through inputs directly as ArrayRef
```

Comment:
Err... can't you just use an `initializer_list` here

Reply:
I need `args` twice below, so I need to either repeat the argument list or have it in a variable on the stack. This is, however, going to become obsolete when I pass through the `stack` directly instead of passing in an `ArrayRef`.